arangod/RocksDBEngine/RocksDBValue.cpp

```diff
@@ -141,6 +151,19 @@ S2Point RocksDBValue::centroid(rocksdb::Slice const& s) {
                  intToDouble(uint64FromPersistent(s.data() + sizeof(uint64_t) * 2)));
 }
 
+replication2::LogTerm RocksDBValue::logTerm(const rocksdb::Slice& slice) {
+  TRI_ASSERT(slice.size() >= sizeof(uint64_t));
+  return replication2::LogTerm(uint64FromPersistent(slice.data()));
+}
+
+replication2::LogPayload RocksDBValue::logPayload(const rocksdb::Slice& slice) {
+  TRI_ASSERT(slice.size() >= 8);
```

Comment:
8 == sizeof(uint64_t)?
```suggestion
  TRI_ASSERT(slice.size() >= sizeof(uint64_t));
```